lib/suggestion-list.coffee

```diff
@@ -190,12 +194,14 @@ class SuggestionList
     if marker = editor.getLastCursor()?.getMarker()
       @activeEditor = editor
       @overlayDecoration = editor.decorateMarker(marker, {type: 'overlay', item: this})
+      @bindToMovementCommands(editor)
```

Comment:
How would you feel about moving `@bindToMovementCommand`/`@movementCommandSubscriptions?.dispose()` into the `addKeyboardInteraction`/`removeKeyboardInteraction` methods? Seems exactly where those two pieces of behavior belong.